### ISSUE TITLE
Training a classifier

### ISSUE THREAD

**Issue by rohansood42** [NaturalNode/natural#359]
Is it possible to train the classifier after classifying.

For eg. - If i have already trained a classifier with a training set and then start working on a test set. I want it to add the prediction to its training set so that it can also be used for classifying new samples.

**Comment by s-a** [comment]
Yes. You can use the `.load` method before adding new docs and then train again.
